operations/tasks/update-node_selector-labels.yml

```diff
@@ -0,0 +1,18 @@
+---
+# delete all predefined node selector labels
+- name: Delete all labels
+  command: >
+    oc label node {{ inventory_hostname }} {{ item }}-
```

Comment:
All `oc` command calls need `environment` to be set.

Reply:
ok also adding tags: oc